src/benchmarks/micro/libraries/System.Numerics.Vectors/Perf_Matrix3x2.cs

```diff
@@ -0,0 +1,110 @@
+                                                              2.1f, 2.2f,
+                                                              3.1f, 3.2f);
+
+        [Benchmark]
+        public Matrix3x2 IdentityBenchmark() => Matrix3x2.Identity;
+
+        [Benchmark]
+        public bool IsIdentityBenchmark() => Matrix3x2.Identity.IsIdentity;
+
+        [Benchmark]
+        public Matrix3x2 AddOperatorBenchmark() => Matrix3x2.Identity + Matrix3x2.Identity;
+
+        [Benchmark]
+        public bool EqualityOperatorBenchmark() => Matrix3x2.Identity == Matrix3x2.Identity;
+
+        [Benchmark]
+        public bool InequalityOperatorBenchmark() => Matrix3x2.Identity != Matrix3x2.Identity;
+
+        [Benchmark]
+        public Matrix3x2 MultiplyByMatrixOperatorBenchmark() => Matrix3x2.Identity * Matrix3x2.Identity;
+
+        [Benchmark]
+        public Matrix3x2 MultiplyByScalarOperatorBenchmark() => Matrix3x2.Identity * 0.5f;
+
+        [Benchmark]
+        public Matrix3x2 SubtractOperatorBenchmark() => Matrix3x2.Identity - Matrix3x2.Identity;
+
+        [Benchmark]
+        public Matrix3x2 NegationOperatorBenchmark() => -Matrix3x2.Identity;
+
+        [Benchmark]
+        public Matrix3x2 AddBenchmark() => Matrix3x2.Add(Matrix3x2.Identity, Matrix3x2.Identity);
+
+        [Benchmark]
+        public Matrix3x2 CreateRotationBenchmark() => Matrix3x2.CreateRotation(PI / 2.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateRotationWithCenterBenchmark() => Matrix3x2.CreateRotation(PI / 2.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarXYBenchmark() => Matrix3x2.CreateScale(1.0f, 2.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarWithCenterBenchmark() => Matrix3x2.CreateScale(1.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarXYWithCenterBenchmark() => Matrix3x2.CreateScale(1.0f, 2.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarBenchmark() => Matrix3x2.CreateScale(1.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromVectorBenchmark() => Matrix3x2.CreateScale(Vector2.UnitX);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromVectorWithCenterBenchmark() => Matrix3x2.CreateScale(Vector2.UnitX, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateSkewFromScalarXYBenchmark() => Matrix3x2.CreateSkew(1.0f, 2.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateSkewFromScalarXYWithCenterBenchmark() => Matrix3x2.CreateSkew(1.0f, 2.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateTranslationFromVectorBenchmark() => Matrix3x2.CreateTranslation(Vector2.UnitX);
+
+        [Benchmark]
+        public Matrix3x2 CreateTranslationFromScalarXY() => Matrix3x2.CreateTranslation(1.0f, 2.0f);
+
+        [Benchmark]
+        public bool EqualsBenchmark() => Matrix3x2.Identity.Equals(Matrix3x2.Identity);
```

Comment:
method names are part of the benchmark ID which is used in the reporting system. This is why we try to keep them short. Could you please remove the `Benchmark` suffix from all the benchmark names?

Since `Equals` method already exists, you can use `EqualsSame` name

Reply:
I did this to match the existing names used by the other numerics tests, which we can't change without breaking tracking in the reporting system.